micropub.php

```diff
@@ -36,6 +36,10 @@
 if ( ! defined( 'MICROPUB_DRAFT_MODE' ) )
     define('MICROPUB_DRAFT_MODE', '0');
 
+// For debugging purposes this will log all incoming queries to the error log
+if ( ! defined( 'MICROPUB_LOG' ) )
+    define('MICROPUB_LOG', '0');
```

Comment:
no need for our own config option, just use `WP_DEBUG[_LOG]`. details:
* http://codex.wordpress.org/Debugging_in_WordPress
* http://www.stumiller.me/sending-output-to-the-wordpress-debug-log/